test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1

```diff
@@ -234,6 +278,11 @@ Describe "Test-Connection" -tags "CI" {
         It "MTUSizeDetect works" -Pending:($env:__INCONTAINER -eq 1) {
             $result = Test-Connection $hostName -MtuSize
 
+            if (-not $?)
+            {
+                Get-Error | Out-String | Write-Host
```

Comment:
Would it be better to use `-Because` with the first `Should` test to have this in the test log?